crates/augurs-forecaster/src/power_transforms.rs

```diff
@@ -0,0 +1,70 @@
+use crate::transforms::box_cox;
+use argmin::core::*;
+use argmin::solver::brent::BrentOpt;
+
+fn box_cox_log_likelihood(data: &[f64], lambda: f64) -> f64 {
+    let n = data.len() as f64;
+    let transformed_data: Vec<f64> = data.iter().map(|&x| box_cox(x, lambda)).collect();
+    let mean_transformed: f64 = transformed_data.iter().copied().sum::<f64>() / n;
+    let variance: f64 = transformed_data
+        .iter()
+        .map(|&x| (x - mean_transformed).powi(2))
+        .sum::<f64>()
+        / n;
```

Comment:
Hmm this will panic if `n` is zero. Feels unlikely to happen but I wonder if we should check that somewhere (maybe further up the call stack)